src/select-menu/docs/examples/SelectMenu-multi.example

```diff
@@ -1,51 +1,49 @@
-    <Manager>
-      {({ setState, state }) => (
-        <SelectMenu
-          title="Select multiple names"
-          options={options}
-          selected={state.selected}
-          onSelect={item => {
-            let selected
-            if (Array.isArray(state.selected)) {
-              selected = [...state.selected, item.value]
-            } else {
-              selected = arrify(item.value)
-            }
-            const selectedItems = selected
-            const selectedItemsLength = selectedItems.length
-            let selectedNames = ''
-            if (selectedItemsLength === 0) {
-              selectedNames = ''
-            }
-            else if (selectedItemsLength === 1) {
-              selectedNames = selectedItems.toString()
-            } else if (selectedItemsLength > 1 ) {
-              selectedNames = selectedItemsLength.toString() + " selected..."
-            }
-            setState({
-              selected,
-              selectedNames
-            })
-          }}
-          onDeselect={item => {
-            const deselectedItemIndex = state.selected.indexOf(item.value)
-            const selectedItems = state.selected.filter(
-              (_item, i) => i !== deselectedItemIndex
-            )
-            const selectedItemsLength = selectedItems.length
-            let selectedNames = ''
-            if (selectedItemsLength === 0) {
-              selectedNames = ''
-            }
-            else if (selectedItemsLength === 1) {
-              selectedNames = selectedItems.toString()
-            } else if (selectedItemsLength > 1 ) {
-              selectedNames = selectedItemsLength.toString() + " selected..."
-            }
-            setState({ selected: selectedItems, selectedNames })
-          }}
-        >
-          <Button>{state.selectedNames || 'Select multiple...'}</Button>
-        </SelectMenu>
-      )}
-    </Manager>
\ No newline at end of file
+<Component
+  initialState={{
+    options: options,
+    selected: [] 
+  }}
+>
+  {({ state, setState }) => (
+    <SelectMenu
+      title="Select multiple names"
+      options={state.options}
+      selected={state.selected}
+      onSelect={item => {
+        let  selected = [...state.selected, item.value]
```

Comment:
Seems this should be `const`?

Reply:
You are correct; the `let` vs. `const` mutability approach was a trip. All that, then I failed on the very copy/paste I commented on. Great catch!